src/main/java/com/ericsson/jenkinsci/hajp/api/PluginsManager.java

```diff
@@ -45,7 +44,8 @@ public void updatePluginConfig(String pluginName, String fileName, byte[] fileAs
         file = new File(rootDir, fileName);
```

Comment:
Remove pluginmanager and processor all together. That feature needs to be completely re-written/re-done.